setup/Directory.Build.props

```diff
@@ -0,0 +1,22 @@
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project>
+  <PropertyGroup>
+    <IsUnitTestProject>false</IsUnitTestProject>
+    <IsIntegrationTestProject>false</IsIntegrationTestProject>
+    <IsTestProject>false</IsTestProject>
+    <OutDirName Condition="'$(OutDirName)' == ''">Dlls</OutDirName>
+  </PropertyGroup>
+
+  <Import Project="..\Directory.Build.props"/>
+  <Import Project="..\build\import\VisualStudio.XamlRules.props" />
```

Comment:
Instead of duplicating this import, can we just move this property in the file into RepoLayout.props and delete this file.